### ISSUE TITLE
Calculate a custom tip: Android Basics with Compose  

### ISSUE THREAD

**Issue by correabuscar** [google-developer-training/basic-android-kotlin-compose-training-tip-calculator#19]
**URL of codelab**
https://developer.android.com/codelabs/basic-android-kotlin-compose-calculate-tip?continue=https%3A%2F%2Fdeveloper.android.com%2Fcourses%2Fpathways%2Fandroid-basics-compose-unit-2-pathway-3%23codelab-https%3A%2F%2Fdeveloper.android.com%2Fcodelabs%2Fbasic-android-kotlin-compose-calculate-tip#4

**In which task and step of the codelab can this issue be found?**
`5. Set an action button`

**Describe the problem**
There's no explanation as to why `KeyboardOptions.Default.copy(..)` became `KeyboardOptions(...)`. If it's because they're equivalent, then what was the point of using the former in the first place? or, perhaps this could've been mentioned, there, that they're the same ? (if they are)

> Use the `KeyboardOptions.Default.copy` function to use the other default options like capitalization and auto correction.

So are the following two equivalent? because that's the only explanation that makes sense to me currently, as to why `KeyboardOptions.Default.copy(..)` became `KeyboardOptions(...)` in the codelab, with no explanation.
```kotlin
@Composable
fun EditNumberField(
   //...
) {
   TextField(
       //...
       keyboardOptions = KeyboardOptions.Default.copy(
           keyboardType = KeyboardType.Number,
           imeAction = ImeAction.Next
       )
   )
}
```
and
```kotlin
@Composable
fun EditNumberField(
   //...
) {
   TextField(
       //...
       keyboardOptions = KeyboardOptions(
           keyboardType = KeyboardType.Number,
           imeAction = ImeAction.Next
       )
   )
}
```

In other words, why isn't `KeyboardOptions.Default.copy(...)`(instead of `KeyboardOptions(...)`) passed as an argument there in steps 5&6, eg. here:
```kotlin
EditNumberField(
   label = R.string.bill_amount,
   keyboardOptions = KeyboardOptions(
       keyboardType = KeyboardType.Number,
       imeAction = ImeAction.Next
   ),
   value = amountInput,
   onValueChange = { amountInput = it }
)
```

From what I can tell, they are equivalent, as per my interpretation of this:
![theyareequivalent](https://user-images.githubusercontent.com/106607441/180367516-551b3af6-e634-436b-ab37-a19236ad3b68.png)

**Comment by android-dev-lxl** [comment]
@correabuscar 
Thank you, this is fixed and to be published.
